planner/core/expression_rewriter.go

```diff
@@ -1057,7 +1108,9 @@ func (er *expressionRewriter) binaryOpToExpression(v *ast.BinaryOperationExpr) {
 		return
 	}
 	er.ctxStack = er.ctxStack[:stkLen-2]
+	er.ctxNameStk = er.ctxNameStk[:stkLen-2]
```

Comment:
May we think about merging the `ctxStack` and `ctxNameStl`? They always do the same operation.

Reply:
I'll try this way.

Reply:
changed